package.json

```diff
@@ -5,15 +5,17 @@
 	"main": "index.js",
 	"scripts": {
 		"format": "prettier --write 'packages/**/*.ts'",
-		"api::build": "npm run build --workspace api; npm run build-frontend --workspace api",
-		"api::package": "npm run package --workspace api",
 		"worker::build": "npm run build --workspace worker; npm run build --workspace worker",
 		"worker::package": "npm run package --workspace worker",
+		"api::build": "npm run build --workspace api",
+		"client::build": "npm run build --workspace client",
```

Comment:
would it make sense to add `client::start` here? Or, I can just add it in this one once you've merged as I'm already added ts-node-dev and api::start